dm/dm/master/server.go

```diff
@@ -405,6 +411,45 @@ func subtaskCfgPointersToInstances(stCfgPointers ...*config.SubTaskConfig) []con
 	return stCfgs
 }
 
+func (s *Server) initClusterID(ctx context.Context) error {
+	log.L().Info("init cluster id begin")
+	ctx1, cancel := context.WithTimeout(ctx, time.Second*10)
+	defer cancel()
+
+	resp, err := s.etcdClient.Get(ctx1, common.ClusterIDKey)
+	if err != nil {
+		return err
+	}
+
+	// New cluster, generate a cluster id and backfill it to etcd
+	if len(resp.Kvs) == 0 {
+		ts := uint64(time.Now().Unix())
+		clusterID := (ts << 32) + uint64(rand.Uint32())
+		clusterIDBytes := make([]byte, 8)
+		binary.BigEndian.PutUint64(clusterIDBytes, clusterID)
```

Comment:
may just use `uuid.New().String()` in `"github.com/google/uuid"`

Reply:
> may just use `uuid.New().String()` in `"github.com/google/uuid"`

Random integer type is needed, so i didn't adopt uuid.